docs/reference/release.md

```diff
@@ -1,21 +1,61 @@
+
+## Release Candidates
+
+When a new version milestone is published in the [Gutenberg milestones](https://github.com/WordPress/gutenberg/milestones), a date should accompany it. We usually track the current and next version milestone, and we tend to release a version every two weeks. The date in the milestone is the date of tagging the release candidate, not the final version that will be published to the plugin repository.
+
+Since [`3.8`](https://github.com/WordPress/gutenberg/releases/tag/v3.8.0), we [publish a release candidate](https://github.com/WordPress/gutenberg/releases/tag/v3.8.0-rc.1) at least one week before we actually publish a new version. This release candidate is available for plugin authors and for users to test. If any bugs/regressions are found with a release candidate, they should be cherry-picked into a new release candidate and it should be published.
+
+Release candidates should be versioned incrementally, starting with `-rc.1`, then `-rc.2`, and so on.
+
+### Cherry-picking into Release Candidates
+
+If a bug is found in a release candidate and a fix is committed to `master`, we should include that fix in a new release candidate. To do this you'll need to use `git cherry-pick`. This way only fixes are added to the release candidate and not all the new code that has landed since tagging.
+
+(This guide assumes we are targetting version `4.0.0`, we released `4.0.0-rc.1` already, and the commit hash for the fix is `123456abcdef`. Substitute those values for your own.)
+
+Here is the process to cherry-pick a fix and push a new tag:
+
+```bash
+git fetch --tags
+git checkout v4.0.0-rc1
+git cherry-pick 123456abcdef # If you want to cherry-pick multiple fixes for a
+                             # new release candidate, you can cherry-pick
+                             # multiple times.
+git tag v4.0.0-rc2
+git push origin v4.0.0-rc2
+```
+
+After you've pushed a new release-candidate for a version, you should update the [GitHub Releases page](https://github.com/WordPress/gutenberg/releases). In this example, you would visit https://github.com/WordPress/gutenberg/releases/edit/v4.0.0-rc.1 and click _"Edit Release"_, then change the tag from `v4.0.0-rc1` to `v4.0.0-rc2`. You should create a new `gutenberg.zip` file, and replace the existing one on that page with the new release.
+
+It's worth mentioning that a new release candidate has been released in the [`#core-editor` channel](https://wordpress.slack.com/messages/C02QB2JS7).
+
+## How To Release Gutenberg
+
+This guide applies to both release candidates and actual releases. The only difference between a release candidate and actual release is that the "push to repository" step is only done for actual releases and not release candidates.
+
+### Writing the Release Post and Changelog
 
 * Open the [recently updated PRs view](https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc) and find the PR where the last version bump occurred.
```

Comment:
This is no longer true because the version bump may have occured after a lot of merges in master not included in the final version. We need to find the first RC version bump instead.